src/nbl/video/device_capabilities/device_features.json

```diff
@@ -49,7 +49,7 @@
                 {
                     "type": "bool",
                     "name": "tessellationShader",
-                    "value": false
+                    "value": true
```

Comment:
look, there's no "expose" (which means it defaults to optional), the value NEEDS to be false becvause thats the default/initializer value (you don't want to enable `robustBufferAccess` unless the user explicitly asks for it